src/components/Header/Heading.js

```diff
@@ -63,7 +64,7 @@ Heading.propTypes = {
     medium: PropTypes.oneOf(Object.keys(typography.size)),
     large: PropTypes.oneOf(Object.keys(typography.size))
   }),
-  weight: PropTypes.oneOf(["light", "regular", "extraBold"]),
+  weight: PropTypes.oneOf(["light", "regular", "extraBold", "semiBold"]),
```

Comment:
Could you please define a constant for this enum?

Reply:
Done.